shuup/front/settings.py

```diff
@@ -9,6 +9,13 @@
 #:
 #: This is the easiest way to customize the order creation process
 #: without having to override a single URL or touch the ``shuup.front`` code.
+
+
+#: Used to get the login form for login dropdown
```

Comment:
Improve this text. And move this to the bottom or top of the code. You placed this in the middle of a existing help text :smile: Check above the comments, it describes the `SHUUP_BASKET_ORDER_CREATOR_SPEC` below.

Reply:
Fixed